boskos/README.md

```diff
@@ -0,0 +1,51 @@
+# WIP boskos
+
+
+## Background
+[βοσκός](https://en.wiktionary.org/wiki/%CE%B2%CE%BF%CF%83%CE%BA%CF%8C%CF%82) - cattle-master in greek!
+
+boskos is a http service that allow k8s test jobs to run independently from GCP projects.
+
+## Introduction
+
+Imagine boskos is a ranch with cattle, and to perform each job in the ranch you need to hold a cattle. 
+
+The ranch is inited with a list of GCP projects, one JSON entry per line, from `-config` flag
+
+Interface:
+
+* `/request?job=FOO&duration=X`: with job FOO, checkout a cattle for duration X.
+X can be in any unit accept by [`time.ParseDuration`](https://golang.org/pkg/time/#ParseDuration)
+* `/return?job=FOO`: Job FOO is done, return the cattle checked out by job FOO.
+* `/sync`: Ranch janitor, clean and restore any timed-out cattles.
```

Comment:
I would prefer we separate out the state of the system (which projects are loaned out and to who?) from how they are used.

Specifically I would like the janitor to a) run in a separate pod and b) have the ability to run in a replicated fashion